ext/AvizExt/viz/scenarios.jl

```diff
@@ -1,8 +1,12 @@
 using JuliennedArrays: Slices
+using ADRIA.analysis: series_confint
+
+include("../../../src/utils/text_display.jl")
```

Comment:
☹️

Reply:
Sorry, that was committed by accident, I'm not even using it...